src/pages/developing/web-components-tutorial/step-1.mdx

```diff
@@ -0,0 +1,766 @@
+}
+```
+
+As per the global actions we will use additional styling to add icons to our
+content switcher.
+
+```scss path=style.scss
+.theme-selector__icon {
+  width: 1.25rem;
+  height: 1.25rem;
+  background-color: $text-primary;
+}
+
+cds-content-switcher-item[selected] .theme-selector__icon {
+  /* switch icon color when selected */
+  background-color: $background;
+}
+
+.theme-selector__icon--light {
+  mask: url('/sun.svg') no-repeat center;
+}
+
+.theme-selector__icon--system {
+  mask: url('/brightness-contrast.svg') no-repeat center;
+}
+
+.theme-selector__icon--dark {
+  mask: url('/moon.svg') no-repeat center;
+}
+```
+
+Currently the theme switcher looks good but needs the following Javascript to
+switch themes.
+
+```javascript path=main.js
+const handleSwitch = (ev) => {
+  // Applies new theme or defers to system preferences by removing theme
+  switch (ev.detail.item.value) {
+    case 'light':
+      bodyEl.classList.remove('g100');
+      bodyEl.classList.add('g10');
+      break;
+    case 'dark':
+      bodyEl.classList.remove('g10');
+      bodyEl.classList.add('g100');
+      break;
+    default:
+      bodyEl.classList.remove('g10');
+      bodyEl.classList.remove('g100');
+  }
+};
+document
+  .querySelector('.theme-selector')
+  .addEventListener('cds-content-switcher-selected', handleSwitch);
+
+const handleHeaderCompliment = (ev) => {
+  document
+    .querySelector('cds-header')
+    .classList.toggle('compliment', ev.target.checked);
+};
+document
+  .querySelector('.theme-header__compliment')
+  .addEventListener('cds-checkbox-changed', handleHeaderCompliment);
+```
+
+At this point our theme switcher is mostly working, only the checkbox
+`Global header reverse theme` appears to do nothing. In the script above the
+class `compliment` is being toggled on and off in the `<header>` tag.
+
+A little more CSS is needed to make this functionality work. Add the following
+in addition to the existing theme classes.
+
+```scss path=style.scss
+:root .compliment {
+  @include theme(themes.$g100);
+
+  @media (prefers-color-scheme: dark) {
+    @include theme(themes.$g10);
+  }
+}
+
+.g10 .compliment {
+  @include theme(themes.$g100);
+}
+
+.g100 .compliment {
+  @include theme(themes.$g10);
+}
+```
+
+Then replace the cds-header tags `g100` class in `index.html` with `compliment`.
+
+```html path=index.html
+<cds-header class="compliment"></cds-header>
+```
+
+### Skip to content
+
+When creating navigation headers, it's important to have a `Skip to content`
+link so keyboard users can skip the navigation items and go straight to the main
+content.
+
+Import th ecomponent in `main.js`
+
+```javascript path=main.js
+import '@carbon/web-components/es/components/skip-to-content/index.js';
+```
+
+Add in to our header in `index.html` as the first child of our `cds-header`
+component.
+
+```html path=index.html
+<cds-header class="compliment">
+  <cds-skip-to-content href="#main-content"></cds-skip-to-content>
```

Comment:
can we add some `...` below to show that we should keep the other content already there